src/main/charts/confluence/templates/configmap-fluentd.yaml

```diff
@@ -8,14 +8,9 @@ metadata:
 data:
   fluent.conf: |
     <source>
-      @type tail
-      path /application-data/logs/atlassian-confluence.log*
-      read_from_head true
-      tag confluence.log
-      <parse>
-        @type regexp
-        expression /^(?<time>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3}) (?<level>[A-Z]+) \[(?<thread>\S+)\] \[(?<logger>\S+)\] (?<method>[\w|\$]+) (?<message>.*)$/
-      </parse>
```

Comment:
We can now remove this insider knowledge of the Confluence log format, now we just accept the JSON event whatever it may be